csharp/ql/src/semmle/code/csharp/dataflow/internal/rangeanalysis/SignAnalysisSpecific.qll

```diff
@@ -48,6 +50,10 @@ private module Impl {
 
   private class BooleanValue = AbstractValues::BooleanValue;
 
+  /**
+   * Gets the value of the expression if it can't be converted to integer, but
```

Comment:
The qldoc for this predicate ought to be the same in both the Java and the C# version.

Reply:
I'll move this to the common part with an alias for java as `class ExprWithPossibleValue = J::Literal;`